src/simulations/modeling/adapter.py

```diff
@@ -487,6 +505,27 @@ def bounds(measurement, uncertainty):
                 }
             )
 
+    for measure in proteomics:
+        try:
+            reaction = model.reactions.get_by_id(
+                "prot_{}_exchange".format(measure["identifier"])
```

Comment:
Prefer f-strings over `.format` for consistency

Reply:
done in commit [`1fe6f9a`](https://github.com/DD-DeCaF/simulations/pull/147/commits/1fe6f9ad49497691b7f7feea0d7a10233907d456)